imports/db/posts/schema.js

```diff
@@ -12,8 +12,8 @@ export default new SimplSchema({
     },
     postType: {
         type: String,
-        allowedValues: _.values(PostTypesEnum),
-        defaultValue: _.values(PostTypesEnum)[0]
+        allowedValues: types,
+        defaultValue: types[0]
```

Comment:
you can import PostTypesEnum from post type enums and use it here

Reply:
Why cannot types?